app/Console/Commands/ReviewSignup.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Signup;
+use Illuminate\Console\Command;
+use Rogue\Services\Three\PostService;
+
+class ReviewSignup extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:reviewsignup {signup} {status} {--admin=}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Review all posts under the given signup ID with the given status';
+
+    /**
+     * The Post service instance.
+     *
+     * @var Rogue\Services\Three\PostService
+     */
+    protected $posts;
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct(PostService $posts)
+    {
+        parent::__construct();
+
+        $this->posts = $posts;
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $signup = Signup::findOrFail($this->argument('signup'));
```

Comment:
Would it make sense to grab it with posts here?

Reply:
I guess it's only one signup so probably easier this way since it's already done!